apps/dalmatiner_db/src/dalmatiner_tcp.erl

```diff
@@ -86,6 +98,11 @@ loop(Socket, Transport, State) ->
             lager:error("[tcp:loop] Error: ~p~n", [E]),
             ok = Transport:close(Socket)
```

Comment:
As far as I recall, we can safely ignore these Elvis warnings about this cowboy module.